Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
<img width="935" alt="Screenshot 2023-06-14 at 1 47 47 AM" src="https://github.com/motoon-eg/healthy/assets/45182214/3a89763e-938b-4071-9670-3fdb662f2bb4">

Proportional width is not needed.

Reply:
Leading space is not needed as well.